apps/api/src/app/topics/use-cases/get-topic/get-topic.use-case.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { EnvironmentId, OrganizationId, TopicEntity, TopicId, TopicRepository, UserId } from '@novu/dal';
+
+import { GetTopicCommand } from './get-topic.command';
+
+import { TopicDto } from '../../dtos/topic.dto';
+
+const mapFromCommandToEntity = (
+  command: GetTopicCommand
+): Pick<TopicEntity, '_id' | '_environmentId' | '_organizationId' | '_userId'> => ({
+  _environmentId: command.environmentId as unknown as EnvironmentId,
+  _id: command.id as unknown as TopicId,
+  _organizationId: command.organizationId as unknown as OrganizationId,
+  _userId: command.userId as unknown as UserId,
```

Comment:
IMO this part `command.environmentId as unknown as EnvironmentId` is a dangerous thing.

Imagine that you've changed the type of the `command.environmentId` to let's say `number`, then here in this line, we won't get any TS error, because TS will cast it to `unknown` and then to `EnvironmentId` and it will be happy. 
But we might not remember that the types of `command.environmentId` and `entity._environmentId` should be in sync and if the `EnvironmentId` will be still a string, it will result in errors in other places that use this function. 

This was just an example, and I intended to point out that the `as unknown as something` could harm us. I would instead use both the same types or declare the fields with simple primitives.

WDYT?